inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -1382,6 +1391,14 @@ export class AnimationCurveEditorComponent extends React.Component<
     }
   }
 
+  getCurrentFrame(frame: number) {

```

Comment:
maybe should be renamed to isCurrentFrame ? as it does not return the current frame but only if we are on it

Reply:
good point. I'll rename it